### ISSUE TITLE
[UI idea] Check accessibility issues

### ISSUE THREAD

**Issue by Felienne** [hedyorg/hedy#4124]
@noelgolding was looking into this, he used a tool called [Wave](https://wave.webaim.org/extension/), it returned errors with aria labels and also some other errors with contrast:

<img width="653" alt="image" src="https://user-images.githubusercontent.com/1003685/224777952-e5b97f92-10ee-4d7c-abce-3d4f304e6808.png">

Would be lovely if someone dove in!

I have labeled this good first issue, since it does not require deep knowledge of the code base

**Comment by CasHil** [comment]
I briefly looked into this using Wave to check what the accessibility is like on the homepage. Checking this for all pages is quite an endeavour, but improving the accessibility of some common components can improve the accessibility on all pages. I think it's good to prioritize these common components first, and then approach this starting with the most visited pages.

On the homepage, the errors and alerts look like this:

![image](https://github.com/hedyorg/hedy/assets/42659828/02ee11fa-ef64-4bf4-9f60-dccad8403a50)

1. Some images are missing alt tags. These are used by screen readers or shown when internet speeds are slow. These should be added, preferably in the selected language, if the image is not purely decorative.
2. There are some texts that do not contrast enough with their backgrounds, mainly the navigation bar and the "Try it" button on the homepage. See the following guide: https://usecontrast.com/guide. Wave shows the Contrast Ratio for texts with their background.

**Comment by Felienne** [comment]
Will be picked up by our new intern
